Buildings/Fluid/CHPs/OrganicRankine/Data/WorkingFluids/Acetone.mo

```diff
@@ -0,0 +1,40 @@
+within Buildings.Fluid.CHPs.OrganicRankine.Data.WorkingFluids;
+record Acetone "Data record for acetone"
+  extends Generic(
+    T = {263.15      ,289.25555556,315.36111111,341.46666667,367.57222222,
+ 393.67777778,419.78333333,445.88888889,471.99444444,498.1       },
+    p = {   5344.99011082,  20650.83356607,  61555.89785486, 151351.56361612,
+  321797.04699474, 612178.54891223,1068684.44840007,1745219.88886921,
```

Comment:
fix the format, code is written once but read hundreds of times.

Reply:
Addressed in 465e0d06a64c594b9e61ceeb54735b6638f93cf1.